### PR TITLE
<CountryFlag>

### PR DESCRIPTION
**Pull request by nerik**
screenshot 2019-02-15 at 14 05 12

### CODE REVIEW

File: src/countryflag/countryflag.js [hunk 1]
```diff
@@ -0,0 +1,26 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import countryflag from 'countryflag'
+
+class CountryFlag extends Component {
+  render() {
+    const { iso2, svg } = this.props
+    const flag = countryflag(iso2)
+    return svg === true || flag.emoji === null ? (
+      <img style={{ width: '18px' }} alt={flag.name} src={flag.svg} />
```
Comment (satellitestudiodesign) [inline src/countryflag/countryflag.js]: Can we change `width: '18px'` for `width: 1em` so the image takes the size of the line where it's included?

File: src/countryflag/countryflag.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    ) : (
+      <span>{flag.emoji}</span>
```
Comment (satellitestudiodesign) [inline src/countryflag/countryflag.js]: Returning a span can create unwanted effects, is it necessary?

File: src/countryflag/countryflag.js (hunk 1 above)
Comment (j8seangel) [reply src/countryflag/countryflag.js]: and why not move the size to a `defaultProp` value ? that way we could customise from outside easily.

File: src/countryflag/countryflag.js (hunk 2 above)
Comment (j8seangel) [reply src/countryflag/countryflag.js]: I agree on having a span if we add this accessibility aspect: https://tink.uk/accessible-emoji/

**Pull request merged**